src/content/guides/partial-payment-extension.mdx

```diff
@@ -7,8 +7,17 @@ nav:
 ---
 
 Partial Payment is an extension to the Payment Request Protocol that allows
-customers to pay a partial amount of a payment request using Centrapay.
-However, it is the responsibility of the integrating POS to ensure payment of the remaining amount.
+the flexibility for customers to utilize different forms of payment. This allows
```

Comment:
"that allows the flexibility for ..." does not read all that well IMO. How about "that allows customers to utilise..."?

Reply:
Done